src/java.base/share/classes/sun/net/www/http/HttpCapture.java

```diff
@@ -54,6 +54,9 @@
  * @author jccollet
  */
 public class HttpCapture {
+    // Although accessing files could result in blocking operations,
+    // HttpCapture is a corner case; there seem no urgent need to convert
+    // this class to using java.util.concurrent.locks at this time.
```

Comment:
The updated patch looks good but I think this comment needs another iteration to ensure that it doesn't confuse future maintainers. You could drop it or else replace it with something simple that says that HttpCapture does blocking I/O operations while holding monitors but it's not a concern because it rarely used.

Reply:
Updated as requested.